### PR TITLE
#165 Bump kotlin and compose compiler

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: lint/build.gradle.kts [hunk 1]
```diff
@@ -5,5 +5,6 @@ plugins {
 dependencies {
 	compileOnly(libs.lint.api)
 	testImplementation(libs.test.kotest.runner)
+	testImplementation(libs.test.lint)
```
Comment (MarekMacko) [inline lint/build.gradle.kts]: Missing dependency resulted in 
```
Cannot access class 'com.android.tools.lint.detector.api.Issue'. Check your module classpath for missing or conflicting dependencies

**Review by michal-kucznerowicz — APPROVED** [review]

**Comment by kmaslowiec** [comment]
@MarekMacko Shouldn't we re-open the issue if we work on it? Also please add the direct link for the issue in PR.

**Comment by MarekMacko** [comment]
@kmaslowiec 
1. Main goal of the #165 task is done, so the task is done as well. This PR is just "extra" work, connected with it. Otherwise, should I create a separate issue for each dependency bump? Feel free to post you objections, if you have any, but I don't see the point for. 
2. What kind of direct link you are referring to? It's already referenced in the commit according to our commit message template

**Comment by kmaslowiec** [comment]
> @kmaslowiec
> 
> 1. Main goal of the [Configure Dependabot #165](https://github.com/intive/Picover/issues/165) task is done, so the task is done as well. This PR is just "extra" work, connected with it. Otherwise, should I create a separate issue for each dependency bump? Feel free to post you objections, if you have any, but I don't see the point for.
> 2. What kind of direct link you are referring to? It's already referenced in the commit according to our commit message template

@MarekMacko I meant to add the link like this:
image
Is 5 seconds and it directly takes us to the issue.

**Comment by MarekMacko** [comment]
> > @kmaslowiec
> > 
> > 1. Main goal of the [Configure Dependabot #165](https://github.com/intive/Picover/issues/165) task is done, so the task is done as well. This PR is just "extra" work, connected with it. Otherwise, should I create a separate issue for each dependency bump? Feel free to post you objections, if you have any, but I don't see the point for.
> > 2. What kind of direct link you are referring to? It's already referenced in the commit according to our commit message template
> 
> @MarekMacko I meant to add the link like this: image Is 5 seconds and it directly takes us to the issue.

You can click on commit message in the same manner, I don't see a point to duplicate it when it's already in place. Is a new agreement or I missed something?
image

**Review by BorysChajdas — APPROVED** [review]

**Comment by MarekMacko** [comment]
@kmaslowiec You've added a thumbs up but no approval is present from your side 😞

**Comment by kmaslowiec** [comment]
> @kmaslowiec You've added a thumbs up but no approval is present from your side 😞

I was not fast enough

**Comment by MarekMacko** [comment]
> > @kmaslowiec You've added a thumbs up but no approval is present from your side 😞
> 
> I was not fast enough

I'm confused, do you approve or not?

**Comment by kmaslowiec** [comment]
> > > @kmaslowiec You've added a thumbs up but no approval is present from your side 😞
> > 
> > 
> > I was not fast enough
> 
> I'm confused, do you approve or not?

It is approved by the others, so I thought I do not need to. Yes. Approved.

**Review by kmaslowiec — APPROVED** [review]

**Comment by MarekMacko** [comment]
> > > > @kmaslowiec You've added a thumbs up but no approval is present from your side 😞
> > > 
> > > 
> > > I was not fast enough
> > 
> > 
> > I'm confused, do you approve or not?
> 
> It is approved by the others, so I thought I do not need to. Yes. Approved.

If discussion has been started, then we should finalize it, regardless of others approvals. Thank you! 🍻

**Pull request merged**